src/main/java/org/folio/search/model/config/LanguageConfigEntity.java

```diff
@@ -0,0 +1,20 @@
+package org.folio.search.model.config;
+
+import java.util.UUID;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "language_config")
+@Entity
+public class LanguageConfigEntity {
+  @Id
+  private UUID id;
```

Comment:
why do we need uuid for the language entity, languageCode can be used as id for the table

Reply:
It is common approach on FOLIO I think, there is at least few more examples where a surrogate key is used instead of natural (locations, institutions (there is a code for each institution), etc.).

@MikhailFokanov, @hjiebsco should we proceed with this suggestion and get rid of UUID identifier and use natural primary key?

Reply:
lang code is cleaner in my view.

Reply:
Ok, I'll use then language code as PK.

Reply:
Igor has confirmed that it is fine from UI point of view do not have UUID. The UUID has been removed.